.gitignore

```diff
@@ -1,5 +1,5 @@
 .idea/
-bin/
+/bin
```

Comment:
what does this change?

Reply:
that was intentional.
When set to `bin/` the folder where `createDocu.groovy` is located is in the exclude list in my IDE ...